src/providers/WorkspaceSymbolProvider.ts

```diff
@@ -1,73 +1,55 @@
 import * as vscode from "vscode";
 import { AtelierAPI } from "../api";
 import { DocumentContentProvider } from "./DocumentContentProvider";
+import { filesystemSchemas } from "../extension";
+import { fileSpecFromURI } from "../utils/FileProviderUtil";
 
 export class WorkspaceSymbolProvider implements vscode.WorkspaceSymbolProvider {
-  private sql: string =
-    "SELECT * FROM (" +
-    "SELECT Name, Parent->ID AS Parent, 'method' AS Type FROM %Dictionary.MethodDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'property' AS Type FROM %Dictionary.PropertyDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'parameter' AS Type FROM %Dictionary.ParameterDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'index' AS Type FROM %Dictionary.IndexDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'foreignkey' AS Type FROM %Dictionary.ForeignKeyDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'xdata' AS Type FROM %Dictionary.XDataDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'query' AS Type FROM %Dictionary.QueryDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'trigger' AS Type FROM %Dictionary.TriggerDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'storage' AS Type FROM %Dictionary.StorageDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'projection' AS Type FROM %Dictionary.ProjectionDefinition" +
-    ") WHERE %SQLUPPER Name %MATCHES ?";
+  private readonly _sqlPreifx: string =
```

Comment:
This is just cosmetic, but @isc-bsaviano did you want this variable to be called `_sqlPrefix` (not Preifx)?

Reply:
Yeah, this is intentional. I chose `_sqlPrefix` to mean "the start of the query that is always used".

Reply:
I was just saying you wrote `ifx` in the code instead of `fix`

Reply:
Good catch, I totally missed that!